components/validator/l2_output_submitter.go

```diff
@@ -78,13 +79,22 @@ func NewL2OutputSubmitter(parentCtx context.Context, cfg Config, l log.Logger, m
 	cCancel()
 
 	cCtx, cCancel = context.WithTimeout(ctx, cfg.NetworkTimeout)
-	defer cCancel()
 	callOpts = utils.NewSimpleCallOpts(cCtx)
 	submissionInterval, err := l2ooContract.SUBMISSIONINTERVAL(callOpts)
 	if err != nil {
 		cancel()
 		return nil, fmt.Errorf("failed to get submission interval: %w", err)
 	}
+	cCancel()
+
+	cCtx, cCancel = context.WithTimeout(ctx, cfg.NetworkTimeout)
+	defer cCancel()
+	callOpts = utils.NewSimpleCallOpts(cCtx)
+	minBondAmount, err := valpoolContract.MINBONDAMOUNT(callOpts)
+	if err != nil {
+		cancel()
+		return nil, fmt.Errorf("failed to get min bond amount: %w", err)
+	}
```

Comment:
I don't think the `minBondAmount` is necessary, because the `bondAmount` is declared in config, and it will try to bond with that amount..